app/Front/Code/ellison_preview.view.php

```diff
@@ -1,4 +1,5 @@
-<?php /** @var \Tempest\View\GenericView $this */ ?>
+<?php  /** @var \Tempest\View\GenericView $this */
```

Comment:
I reported this formatting issue and will hopefully update Mago before we merge this PR

Reply:
fixed, and released as 0.19.4.